_pytest/skipping.py

```diff
@@ -245,7 +246,12 @@ def pytest_runtest_makereport(item, call):
                 rep.outcome = "skipped"
                 rep.wasxfail = evalxfail.getexplanation()
         elif call.when == "call":
-            rep.outcome = "failed"  # xpass outcome
+            strict_default = item.config.getini('xfail_strict')
```

Comment:
Hmm I think this is the more correct implementation of "strict xfail" than the one I did... with this, I think we don't even need `check_strict_xfail` (which is used in `pytest_pyfunc_call`) in this module anymore at all, because failure/passing will be handled here which seems more appropriate now. :smile: